website/src/app/api.service.ts

```diff
@@ -234,6 +258,16 @@ export class ApiService {
     )
   }
 
+  setIssueStatus(issueID: string, status: string): Observable<Issue> {
+    return this.http.put<Issue>(`/api/v1/issues/${issueID}/setStatus`, undefined, {
+      params: new HttpParams().set('status', status)
+    }).pipe(
+      tap(data => console.log("set status", data)),
+      catchError(this.handleError<Issue>("setIssueStatus", undefined)),
```

Comment:
It looks like on error here we are just console logging the error. We need to make all of these errors popups.